src/one_dragon/utils/reg_utils.py

```
@@ -0,0 +1,76 @@
+        :return: 是否写入成功
+        """
+        try:
+            with winreg.CreateKeyEx(self._root, self._subkey, 0,
+                                    winreg.KEY_READ | winreg.KEY_WRITE) as key:
+                # 备份现有值
+                self._backup = _read_values(key, set(values.keys()))
+
+                # 写入新值
+                for name, (data, reg_type) in values.items():
+                    winreg.SetValueEx(key, name, 0, reg_type, data)
+            return True
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)